README.md

```diff
@@ -88,27 +90,30 @@ Flags:
       --endpoint-type="public"   openstack endpoint type to use (i.e: public, internal, admin)
       --collect-metric-time      time spent collecting each metric
   -d, --disable-metric= ...      multiple --disable-metric can be specified in the format: service-metric (i.e: cinder-snapshots)
-      --disable-service.network  Disable the network service exporter
-      --disable-service.compute  Disable the compute service exporter
-      --disable-service.image    Disable the image service exporter
-      --disable-service.volume   Disable the volume service exporter
-      --disable-service.identity
-                                 Disable the identity service exporter
-      --disable-service.object-store
-                                 Disable the object-store service exporter
-      --disable-service.load-balancer
-                                 Disable the load-balancer service exporter
-      --disable-service.container-infra
-                                 Disable the container-infra service exporter
-      --disable-service.dns      Disable the dns service exporter
-      --disable-service.baremetal
-                                 Disable the baremetal service exporter
-      --disable-service.gnocchi  Disable the gnocchi service 
```

Comment:
why was this removed?

Reply:
This was a mistake not to re-add it when backward compatibility has been added back.
it will be corrected

Reply:
OK